crates/manager/src/executor/event_handler.rs

```diff
@@ -94,7 +97,7 @@ impl VerifiedBlueprint {
                             );
                         }
                         Status::Running => {
-                            error!("Process did not terminate successfully, aborting...");
+                            error!("Received unexpected status: {status:?}, aborting process...");
```

Comment:
```suggestion
                            error!("Process did not terminate successfully, aborting...");
```

I feel the new message is less helpful to an operator.